src/test/kotlin/org/rust/ide/typing/RustTypingEmptyFileFuzzyTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.rust.ide.typing
+
+import com.intellij.openapi.util.text.StringUtil
+
+class RustTypingEmptyFileFuzzyTest : RustTypingTestCaseBase() {
+    override val dataPath: String = ""
+
+    fun testFuzzy() {
+        // These chars cause problems or require special treating
+        val blacklist = setOf('\b', '\t', '\r', '"', '(', '[', '{')
```

Comment:
`\b` and `\r` are rather specific cases
`\t`'s behaviour depends on user configuration
and the rest are chars we handle with our own typing handlers, and are tested in applicable places